@shared/api/internal.ts

```diff
@@ -310,65 +311,44 @@ export const confirmPassword = async (
   return response;
 };
 
-export const getAccountBalances = async ({
-  publicKey,
-  networkDetails,
-}: {
-  publicKey: string;
-  networkDetails: NetworkDetails;
-}): Promise<AccountBalancesInterface> => {
-  const { networkUrl, networkPassphrase } = networkDetails;
-
-  let balances: any = null;
-  let isFunded = null;
-  let subentryCount = 0;
-
+export const getAccountIndexerBalances = async (
+  pubKey: string,
```

Comment:
super super NIT: we tend to refer to public keys as `publicKey` in this repo. Would you mind using that var name here to keep it consistent? I think it makes it easier when var naming conventions are the same throughout the app

Reply:
oh yup, will do!

Reply:
fixed in 12547c76cb7701443423ea42cb16c0f825ec746c